internal/hmyapi/types.go

```diff
@@ -22,11 +22,43 @@ type RPCTransaction struct {
 	To               *common.Address `json:"to"`
 	TransactionIndex hexutil.Uint    `json:"transactionIndex"`
 	Value            *hexutil.Big    `json:"value"`
+	FromShardID      uint32          `json:"fromShardID"`
```

Comment:
Keep it consistent with Transaction field "ShardID"

Reply:
Would that be confusing to the user?

Reply:
We should be consistent . It will be more confusing if not consistent

Reply:
OK